client/src/initial/landingpagetanu/tanulanding.scss

```diff
@@ -0,0 +1,120 @@
+.page-container {
+  background: repeating-linear-gradient(35deg, #c572f8 0px 2px, #b954f8 0px 4px);
+
+  top: 0;
+  height: 100vh;
+}
+.title-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  padding-top: 8%;
+  .under-construction-text {
+    color: rgb(48, 47, 47);
+    font-style: bold;
+    text-align: center;
+    font-size: 80px;
+
+    background: repeating-linear-gradient(145deg, #ecf023 0px 8px, #f3f70d 0px 12px);
+    padding: 20px;
+    margin-bottom: 24px;
+    text-shadow: 0px 2px 4px #000000;
+  }
+  .subtitle-text {
```

Comment:
after changing the colours to the branding guideline schemes... one suggestion is to try using white font instead, `font-color: var(--white)` or use `var(--text-light)`, the dark font is a little overwhelming with the saturation of the other colours